backend/src/test/kotlin/org/loculus/backend/controller/submission/GetReleasedDataEndpointTest.kt

```diff
@@ -114,13 +108,34 @@ class GetReleasedDataEndpointTest(
     fun `GIVEN no sequence entries in database THEN returns empty response & etag in header`() {
         val response = submissionControllerClient.getReleasedData()
 
-        val responseBody = response.expectNdjsonAndGetContent<ProcessedData<GeneticSequence>>()
+        val responseBody = response.expectNdjsonAndGetContent<ReleasedData>()
         assertThat(responseBody, `is`(emptyList()))
         response.andExpect(status().isOk)
             .andExpect(header().string(ETAG, notNullValue()))
             .andExpect(header().string("x-total-records", `is`("0")))
     }
 
+    @Test
+    fun `Given released data THEN does not have unknown top-level fields`() {
```

Comment:
This feels (to someone like me who doesn't know much) like it should be the job of the typechecker rather than a test - but I assume that's not possible

Reply:
I wanted to have an explicit check to avoid having new fields being "leaked" without noticing, as happened previously, where we added a field to a class and, because the tests reused the class when parsing the response, the tests didn't fail.